superset/forms.py

```diff
@@ -10,14 +10,20 @@
 from flask_wtf.file import FileAllowed, FileField, FileRequired
 from wtforms import (
     BooleanField, IntegerField, SelectField, StringField)
+from wtforms.ext.sqlalchemy.fields import QuerySelectField
 from wtforms.validators import DataRequired, NumberRange, Optional
 
-from superset import app
+from superset import app, db
 
 config = app.config
 
 
 class CsvToDatabaseForm(DynamicForm):
+    # pylint: disable=E0211
+    def all_db_items():
+        from superset.models import core as models
```

Comment:
Why are these imports not global?

Reply:
I wrongly assumed it would cause cyclical imports. I moved it to be global